source/_integrations/sensor.markdown

```diff
@@ -47,6 +47,7 @@ The type of data a sensor returns impacts how it is displayed in the frontend. T
 - **pressure**: Pressure in Pa, kPa, hPa, bar, cbar, mbar, mmHg, inHg or psi
 - **reactive_power**: Reactive power in var
 - **signal_strength**: Signal strength in dB or dBm
+- **speed**: Generic speed
```

Comment:
@epenet Actually, I just realized I merged this without units, could you update it to include the units?

Reply:
Opened #24265